lib/honeybadger/version.rb

```diff
@@ -1,4 +1,4 @@
 module Honeybadger
   # The current String Honeybadger version.
-  VERSION = '5.10.2'.freeze
+  VERSION = '5.11.0.beta1'.freeze
```

Comment:
Since we are going to YOLO this, we can let release-please change the version for us.

```suggestion
  VERSION = '5.10.2'.freeze
```